modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItSetOpTest.java

```diff
@@ -141,57 +141,51 @@ public void testExceptNested(SetOpVariant setOp) {
     }
 
     @Test
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-18475")
     public void testSetOpBigBatch() {
         sql("CREATE TABLE big_table1(key INT PRIMARY KEY, val INT)");
         sql("CREATE TABLE big_table2(key INT PRIMARY KEY, val INT)");
 
         int key = 0;
 
-        for (int i = 0; i < 5; i++) {
+        for (int i = 0; i < 3; i++) {
```

Comment:
the same test just a bit light weight, previous version is passed too but too slow